node/_events.mjs

```diff
@@ -707,22 +711,19 @@ EventEmitter.prototype.rawListeners = function rawListeners(type) {
  * @param {string | symbol} type
  * @returns {number}
  */
-EventEmitter.listenerCount = function (emitter, type) {
+export function listenerCount(emitter, type) {
   if (typeof emitter.listenerCount === "function") {
     return emitter.listenerCount(type);
   }
-  return listenerCount.call(emitter, type);
-};
-
-EventEmitter.prototype.listenerCount = listenerCount;
-
+  return _listenerCount.call(emitter, type);
+}
 /**
  * Returns the number of listeners listening to event name
  * specified as `type`.
  * @param {string | symbol} type
  * @returns {number}
  */
-function listenerCount(type) {
+function _listenerCount(type) {
```

Comment:
how about making this like
```ts
const _listenerCount = function listenerCount(type) { .... }
```
I guess that would avoid the change in test case below

Reply:
good idea!